CONTRIBUTING.rst

```diff
@@ -21,15 +21,26 @@ Report a `new issue <https://github.com/mozilla/pontoon/issues/new>`_:
 Docker
 ======
 
-Everything runs in a Docker container. Thus Pontoon requires fewer things to get
-started and you're guaranteed to have the same setup as everyone else and it
-solves some other problems, too.
+While the front-end (JavaScript) build and tests use the host environment for development,
```

Comment:
When you run `make build-frontend` or `make test-frontend`, that still calls into Docker. There's also a `frontend` container still. Should we elaborate on that to avoid confusion?

Or is that too likely to change right now to be documented? :)

Reply:
I already have that fixed in another branch. Figured it'd be okay to have this short intermediate step where the docs are only technically correct.